mlflow/deployments/mlflow/__init__.py

```diff
@@ -130,7 +161,24 @@ def get_endpoint(self, endpoint) -> "Endpoint":
     @experimental
     def list_endpoints(self, page_token=None) -> "PagedList[Endpoint]":
         """
```

Comment:
@harupy I think that `list_endpoints()` should consume the pagination token itself so that users don't have to write this code. Can we make that update?

Reply:
Ok, I'll file a PR.

Reply:
@dbczumar https://github.com/mlflow/mlflow/pull/10500

Reply:
Filed https://github.com/mlflow/mlflow/pull/10501